### PR TITLE
Fix for front link. Not sure what happened?

### PR DESCRIPTION
**Pull request by markmandel**
Front page has `What is Agones? An open source, batteries-included, mulitplayer dedicated game server scaling and orchestration platform that can run anywhere [Kubernetes](https://kubernetes.io) can run.` displaying :man_facepalming:

### CODE REVIEW

**Comment by roberthbailey** [comment]
It's strange that the opencensus link works fine as markdown but the k8s one doesn't. 

There aren't many snapshots of agones.dev in the waybackmachine, but from what I can tell this has been broken for at least a few weeks (e.g. not new with the 0.10 release).

**Comment by markmandel** [comment]
I wonder if it was when I upgraded hugo a while back - and it had this strange effect? :man_shrugging:

**Comment by roberthbailey** [comment]
lgtm

also, needs rebase

File: site/content/en/_index.html [hunk 1]
```diff
@@ -22,8 +22,8 @@
 {{% blocks/lead color="secondary" %}}
 <strong>What is Agones?</strong>
 
-An open source, batteries-included, multiplayer dedicated game server scaling and
-orchestration platform that can run anywhere [Kubernetes](https://kubernetes.io) can run.
+An open source, batteries-included, mulitplayer dedicated game server scaling and
```
Comment (pooneh-m) [inline site/content/en/_index.html]: I undid this change when merged. Not sure how to fix it.

**Comment by markmandel** [comment]
Not quite sure why this is failing. Fixing locally and force pushing back up.

**Comment by pooneh-m** [comment]
> Not quite sure why this is failing. Fixing locally and force pushing back up.

It seems that the failure is a bug that is pushed in. All PRs are failing.

**Comment by markmandel** [comment]
I restarted consul, see if that is the issue. Very confusing, because there doesn't seem to be anything in the logs indicating what is the actual issue. Consul is the only thing I could think of.

**Comment by markmandel** [comment]
So I have seen this specific error in a few e2e tests failing

```
--- FAIL: TestAllocator (37.12s)
    allocator_test.go:74: 
        	Error Trace:	allocator_test.go:74
        	Error:      	Expected nil, but got: &url.Error{Op:"Post", URL:"https://34.83.164.122:443/v1alpha1/gameserverallocation", Err:(*net.OpError)(0xc00062f590)}
        	Test:       	TestAllocator
```

I'm wondering if there is a race condition in there - or maybe there should be a backoff.

**Comment by markmandel** [comment]
Adding a test for flakiness,  and seeing if I can find the root cause over here:
https://github.com/GoogleCloudPlatform/agones/pull/789

**Pull request merged**